### PR TITLE
Add another default route matching rule

### PR DESCRIPTION
**Pull request by cbbfcd**
[#101 ](https://github.com/molefrog/wouter/issues/101), we hava a size trouble, +5b～

### CODE REVIEW

File: test/switch.test.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ it("ignores other elements", () => {
   const rendered = result.children[0].children;
 
   expect(rendered.length).toBe(1);
-  expect(rendered[0].type).toBe(Route);
+  expect(rendered[0].type).toBe('b');
```
Comment (molefrog) [inline test/switch.test.js]: What about this? Looks like it changes the behaviour for non-Route elements/components. Should we just ignore this or keep the backward compatibility? 🤔

Comment (cbbfcd) [reply test/switch.test.js]: I think the current behavior is in line with expectations, it should be the behavior that developers need to control, and I observe that react-router also maintains consistent behavior.

Comment (molefrog) [reply test/switch.test.js]: Ok, can we replace the test case with:
```js

it("matches regular components as well", () => {
  const Dummy = props => props.children;

  const result = testRouteRender(
    "/",
    <Switch>
      <Dummy path="/">Component</Dummy>
      <b>Bold</b>
    </Switch>
  );

  const rendered = result.children[0].children;

  expect(rendered.length).toBe(1);
  expect(rendered[0].type).toBe(Dummy);
});
```

File: test/switch.test.js [hunk 2]
```diff
@@ -113,3 +113,41 @@ it("always ensures the consistency of inner routes rendering", async () => {
 
   unmount();
 });
+
+it("default route via position-dependent with wildcard expression", async () => {
```
Comment (molefrog) [inline test/switch.test.js]:
```suggestion
it("supports catch-all routes with wildcard segments", async () => {
```

Comment (molefrog) [reply test/switch.test.js]: Just to make it more aligned with the "it ..." phrase.

File: test/switch.test.js [hunk 3]
```diff
@@ -113,3 +113,41 @@ it("always ensures the consistency of inner routes rendering", async () => {
...
+    </Switch>
+  );
+
+  const rendered = result.children[0].children;
+
+  expect(rendered.length).toBe(1);
+  expect(result.findByType("h2")).toBeTruthy();
+});
+
+it("default route via position-dependent and without a path prop", async () => {
```
Comment (molefrog) [inline test/switch.test.js]:
```suggestion
it("uses a route without a path prop as a fallback", async () => {
```

**Comment by cbbfcd** [comment]
👌

**Comment by molefrog** [comment]
@cbbfcd Nice job! I'm going to merge this and we will have this feature in the next release!

**Comment by cbbfcd** [comment]
@molefrog Thanks for your CR, Help me a lot. 😄

**Pull request merged**